db/migrate/20230309002853_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.references :user, null: false, foreign_key: true
+
```

Comment:
- [ ] Please create this reference between the users table and the posts table as `author_id`, not `user_id` as you did here.
[Hint] you can do it like that:
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```
![image](https://user-images.githubusercontent.com/68796232/223897637-af412a4a-7160-400e-9d0e-7f5fd8489f83.png)

Reply:
resolved! ✔️